cpp/src/parquet/CMakeLists.txt

```diff
@@ -414,6 +414,7 @@ if(PARQUET_REQUIRE_ENCRYPTION)
                    encryption/test_encryption_util.cc)
   add_parquet_test(encryption-key-management-test
                    SOURCES
+                   encryption/encryption_internal_test.cc
```

Comment:
I would add this to `encryption-test` rather, since this isn't related to key management?

Reply:
Yes sorry I missed that there was a separate `encryption-test` target